lib/components/ToggleButton/style.scss

```diff
@@ -0,0 +1,70 @@
+@import '../../theme/tokens/facade';
+
+@mixin isDisabledState() {
+	&[disabled],
+	&[aria-disabled='true'] {
+		cursor: not-allowed;
+
+		&:not(.loading) {
+			@content;
+		}
+	}
+}
+
+$handleSize: 24px;
+$handleOffset: 2px;
+$borderSize: 1px;
+
+.root {
+	font-size: var(--global--fontSize--5);
+	line-height: var(--global--fontSize--5);
+	background-color: getColour(gray, 100);
+	border-radius: $handleSize + $handleOffset;
+	border: solid #{$borderSize} getColour(gray, 300);
```

Comment:
The _official_ css1 spec, says that the width comes before the style.

[border: border-width border-style border-color|initial|inherit;](https://www.w3schools.com/cssref/pr_border.asp)

> But then again, I wonder my style-lint didn't pick up on this, maybe because scss is doing something funny with that `#{}` thing. 🤷🏼‍♂️